lib/fathead/mdn_css/fetch.pl

```diff
@@ -32,14 +33,21 @@
 my $current_active_connections = 0;
 my $maximum_active_connections = 4;
 my @keyword_urls;
+
+#hash to store properties whose urls redirect to other properties
+my %redirect_map;
+
 queue_urls_for_download();
 
 #see http://mojolicious.org/perldoc/Mojo/IOLoop#recurring
 Mojo::IOLoop->recurring(
     0 => sub {
         for ( $current_active_connections + 1 .. $maximum_active_connections ) {
-            return ( $current_active_connections or Mojo::IOLoop->stop )
-              unless my $url = shift @keyword_urls;
+            my $url = shift @keyword_urls;
+            if(!$url) {
+                DumpFile('redirect_urls.yml', { redirects => {%redirect_map} } );
```

Comment:
Thanks @hchienjo 👍 I had noticed the dumping was happening more than once but was unable to figure where to put it 😄 I have moved it to the bottom now.